experiment_bayes.py

```diff
@@ -48,7 +48,7 @@ def forgetting_b():
     pretrained = "results/domain_a"
     rotation = 0
     # add argument for not training again, only validate
-    is_training = False
+    training = False
```

Comment:
"is_training" is more pythonic, so better to stick to that.

Reply:
but I don't understand what you need this argument for. If you do need it, it would be better to freeze the weights so we can avoid all the if statements